src/main/scala/com/rbmhtechnology/calliope/serializer/SequencedMessageSerializer.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2015 - 2017 Red Bull Media House GmbH <http://www.redbullmediahouse.com> - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rbmhtechnology.calliope.serializer
+
+import akka.actor.ExtendedActorSystem
+import akka.serialization.Serializer
+import com.rbmhtechnology.calliope.SequencedMessage
+import com.rbmhtechnology.calliope.serializer.SequencedMessageFormats.SequencedMessageFormat
+
+import scala.util.Try
+
+class DelegatingSequencedMessageSerializer(system: ExtendedActorSystem)
+  extends SequencedMessageSerializer(system, new DelegatingStringManifestPayloadSerializer(system))
+
+abstract class SequencedMessageSerializer(system: ExtendedActorSystem, payloadSerializer: PayloadSerializer) extends Serializer {
```

Comment:
Shall we use a serializer with support for optional`SequencedMessage` wrappers i.e. delegate to another serializer if the manifest doesn't indicate a sequenced message? We could follow this pattern for a more general protocol stack i.e. delegate to lower level if current one doesn't apply.